### DISCUSSION TITLE
Dialegt

### DISCUSSION THREAD

**Discussion by lionardben211** [deepgram/community#810] [Feature Requests]
I think if you focus on a dialect instead of a language, you will have something unique that nobody else in the market has, and you will be far ahead of your competitors like NVIDIA. I suggest you start with the Moroccan dialect

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by SandraRodgers** [comment]
Thank you for this suggestion! I have passed it on to our product team for consideration.
